manticore/models/query.py

```diff
@@ -25,6 +27,9 @@ def match(self, *args, **kwargs):
     def options(self, **kwargs):
         """ Adds OPTIONS clause to search query."""
         qs: SearchQuerySet = self._clone()
+        for key, value in kwargs.items():
+            if isinstance(value, dict):
```

Comment:
Once again. option "index_weights" contains index names, not field names. Calling `_check_field_models` on each dict makes "index_weights" option unusable.

Reply:
I think I finally understand what you mean. Fixed